src/speeds.jl

```diff
@@ -1,3 +1,27 @@
+export Alfven_velocity, Alfven_speed
+
+function Alfven_velocity(B::BField, ρ::Density)
+    return B / sqrt(μ0 * ρ) |> upreferred
+end
+
+function Alfven_velocity(B::BField, n::NumberDensity; mass_numb = 1, Z = 1)
+    ρ = n * (mass_numb * mp + Z * me)
```

Comment:
We should consider not including the effect of the electrons, since that is likely what most people will expect when they ask for this.

Also, I would prefer if this was defined in terms of ion mass with a fallback to mass number.

Reply:
I think you are right in the context of velocity . I will update it later.